game/src/main/kotlin/world/gregs/voidps/world/activity/skill/fletching/Bolts.kts

```diff
@@ -0,0 +1,48 @@
+package world.gregs.voidps.world.activity.skill.fletching
+
+import world.gregs.voidps.engine.client.message
+import world.gregs.voidps.engine.client.ui.interact.itemOnItem
+import world.gregs.voidps.engine.data.definition.data.FletchBolts
+import world.gregs.voidps.engine.entity.character.player.chat.ChatType
+import world.gregs.voidps.engine.entity.character.player.skill.Skill
+import world.gregs.voidps.engine.entity.character.player.skill.level.Level.has
+import world.gregs.voidps.engine.entity.item.Item
+import world.gregs.voidps.engine.inv.inventory
+import world.gregs.voidps.engine.inv.transact.operation.AddItem.add
+import world.gregs.voidps.engine.inv.transact.operation.RemoveItem.remove
+
+itemOnItem("feather", "*_bolts_unf") {
+    val boltUsed = if (toItem.boltsUnf) toItem else fromItem
```

Comment:
I'll merge in #520 first which includes a change (b2a35f1a584def546e9787b53477797ab4611bf3) which means items in `itemOnItem` will always be in the same order as the parameters so you don't have to do this in the future.

Reply:
#520 is merged, if you rebase this and the related/similar code can be removed